src/components/bot-challenge-protection/index.jsx

```diff
@@ -22,14 +22,13 @@ const BotChallengeProtection = ({
 	const { isInitialized } = useIdentity();
 	const { recaptchaVersion, siteKey, isRecaptchaEnabled } = useRecaptcha(challengeIn);
 
-	const recaptchaRef = React.createRef();
+	const recaptchaRef = useRef();
 
 	useEffect(() => {
 		if (isRecaptchaEnabled && recaptchaVersion === RECAPTCHA_V2 && (captchaError || error)) {
-			/* eslint-disable-next-line */
 			recaptchaRef.current.reset();
 		}
-	}, [captchaError, error, isRecaptchaEnabled, recaptchaRef, recaptchaVersion, resetRecaptcha]);
+	}, [captchaError, error, isRecaptchaEnabled, recaptchaVersion, resetRecaptcha, recaptchaRef]);
 
 	const onChange = (value) => {
```

Comment:
This `onChange` function is getting called out as uncovered code in the test run for this PR. The coverage is being counted because of the changes above. You can add an `/* istanbul ignore next */` before this function since it's just a pass-through to a third-party component. That should be good enough to get the coverage up to the 90% minimum.